binplace.targets

```diff
@@ -10,21 +10,35 @@
     <BinPlaceStuffDependsOn Condition="'$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceReferenceAssembly</BinPlaceStuffDependsOn>
     <BinPlaceStuffDependsOn Condition="'$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <BinplaceConfiguration Include="$(BuildConfiguration)">
+      <RefPath>$(RefPath)</RefPath>
+      <RuntimePath>$(RuntimePath)</RuntimePath>
+    </BinplaceConfiguration>
+    <BinplaceConfiguration Condition="'$(IsNETCoreApp)' == 'true'" Include="netcoreapp-$(OSGroup)">
+      <RefPath Condition="'$(IsNETCoreAppRef)' == 'true'">$(BinDir)netcoreapp\pkg\ref</RefPath>
+      <RuntimePath>$(BinDir)netcoreapp\pkg\lib</RuntimePath>
+    </BinplaceConfiguration>
+    <BinplaceConfiguration Include="netstandard-$(OSGroup)">
+      <RefPath>$(RefRootPath)netstandard</RefPath>
+    </BinplaceConfiguration>
+  </ItemGroup>
```

Comment:
Is it your expectation that we will add more of these configurations for uap and netfx?